cellpack/tests/test_data_doc.py

```diff
@@ -0,0 +1,36 @@
+from cellpack.autopack.DBRecipeHandler import DataDoc
+from cellpack.tests.mocks.mock_db import MockDB
+
+mock_db = MockDB({})
+
+object_example = {
+    "count": 121,
+    "object": {
+        "gradient": {
+            "mode": "surface",
+            "name": "nucleus_surface_gradient",
+        },
+        "name": "peroxisome",
+    },
+}
+
+composition_example = {
+    "count": 1,
+    "regions": {"interior": []},
+    "object": {
+        "name": "mean_nucleus",
+        "partners": {"all_partners": []},
+    },
+    "name": "nucleus",
+}
+
+
+def test_is_nested_list():
+    assert DataDoc.is_nested_list([]) is False
+    assert DataDoc.is_nested_list([[], []]) is True
+    assert DataDoc.is_nested_list([[1, 2], [3, 4]]) is True
```

Comment:
Should `DataDoc.is_nested_list([1, [1, 2]])` be `True` or `False`?

Reply:
Good thought! Currently it returns False, though it should return True. While we don't typically encounter such nested lists like [1, [1, 2]] in our recipe data, but you are right that our method should be robust enough to handle all types of nested lists. I just modified the function to run a loop, should be good this time.